docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[running-on-kubernetes-standalone]]
 = Run {agent} Standalone on Kubernetes
 
-:manifest: https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml
+:manifest: https://raw.githubusercontent.com/elastic/elastic-agent/{bare_version}/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml
```

Comment:
```suggestion
:manifest: https://raw.githubusercontent.com/elastic/elastic-agent/v{bare_version}/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml
```

Reply:
The tags in the repos use `v<version>` format: https://github.com/elastic/elastic-agent/tags

Reply:
Thanks for catching that!